sdk/contentsafety/azure-ai-contentsafety/src/samples/README.md

```diff
@@ -0,0 +1,71 @@
+---
+page_type: sample
+languages:
+    - java
+products:
+    - azure
+    - azure-cognitive-services
+    - azure-cognitive-services
+urlFragment: ai-contentsafety-samples
+---
+
+# Azure AI Content Safety Samples client library for Java
+
+This document explains samples and how to use them.
+
+## Examples
+
+Following section document various examples.
+
+### Create Content Safety Client Sample
+
+* [CreateContentSafetyClient.java][sample_CreateContentSafetyClient] - Contains samples for following scenarios:
+    * Authenticate client by Key Credential
+    * Authenticate client by Token Credential
+  
+### Analyze Text Sample
+
+* [AnalyzeText.java][sample_AnalyzeText] - Contains samples for following scenarios:
+    * Authenticate client
+    * Analyze text
+
+### Analyze Image Samples
+
+* [AnalyzeImage.java][sample_AnalyzeImage] - Contains samples for following scenarios:
+    * Authenticate client
+    * Analyze image
+
+### Manage text blocklist Samples
+
+* [ManageTextBlocklist.java][sample_ManageTextBlocklist] - Contains samples for following scenarios:
+    * Authenticate client
+    * Create or update text blocklist
+    * Add blockItems
+    * Analyze text with blocklists
+    * List text blocklists
+    * Get text blocklist
+    * List blockItems
+    * Get blockItem
+    * Remove blockItems
+    * Delete text blocklist
+
+## Troubleshooting
+
+### General
+
+|Error Code	|Possible reasons	|Suggestions|
```

Comment:
The name `Error Code` is a bit confusing because I would thought it is error code like `404, 500` etc..  might be better to name it as `Error Type`?

Reply:
It is a filed defined in our error response body